programs/clearing_house/src/math/fees.rs

```diff
@@ -124,6 +125,8 @@ fn calculate_taker_fee(quote_asset_amount: u64, fee_tier: &FeeTier) -> ClearingH
         .cast::<u128>()?
         .checked_mul(fee_tier.fee_numerator as u128)
         .ok_or_else(math_error!())?
+        .checked_add((fee_tier.fee_denominator / 2) as u128)
```

Comment:
why do you do this instead of just adding by one or doing div_ceil?

Reply:
good to do, will charge them a lil more (not just round up .5)